packages/runtime-vapor/src/apiCreateVaporApp.ts

```diff
@@ -8,10 +8,12 @@ import { warn } from './warning'
 import { version } from '.'
 import { render, setupComponent, unmountComponent } from './apiRender'
 import type { RawProps } from './componentProps'
+import type { Slots } from './componentSlots'
 
 export function createVaporApp(
   rootComponent: Component,
   rootProps: RawProps | null = null,
+  rootSlots: Slots | null = null,
```

Comment:
In runtime-core, `createApp` only takes two parameters. Similarly, `createVaporApp` should follow this pattern.